build/BuildPackageCommand.php

```diff
@@ -54,7 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $filesystem->mkdir($buildDir, 0755);
         $progress->advance();
 
-        $filesystem->mirror($sourceDir, "${buildDir}/${name}");
+        $filesystem->mirror(/** @scrutinizer ignore-type */ $sourceDir, "${buildDir}/${name}");
```

Comment:
I'm not sure I want to fill up the codebase with directions to scrutinizer. Is this really needed? Could't we just teach scrutinizer to ignore whatever the problem is?

Reply:
Removed it.